index.html

```diff
@@ -336,45 +336,38 @@ <h3>
-        the
-        <a data-cite="resource-timing-2#dfn-timing-allow-check">timing allow
-          check</a> algorithm, the <dfn>unloadEventStart</dfn> attribute MUST
-        return a {{DOMHighResTimeStamp}} with a
-        time value equal to the time immediately before the user agent starts
-        the <a data-cite="HTML/browsing-the-web.html#unloading-documents">unload event</a> of
-        the previous document.
-      </p>
-      <p>
-        If there is no previous document or the above <a data-cite="resource-timing-2#dfn-timing-allow-check">timing
-          allow check</a>
-        algorithm does not pass, this attribute MUST return a <a
-          data-cite="hr-time-2#dom-domhighrestimestamp">DOMHighResTimeStamp</a> with a
-        time value equal to zero.
-      </p>
-      <p data-dfn-for='PerformanceNavigationTiming'>
-        If the previous document, and all <a data-cite="FETCH#redirect-status">HTTP redirects</a> when navigating, pass
-        the
-        <a data-cite="resource-timing-2#dfn-timing-allow-check">timing allow
-          check</a> algorithm, the <dfn>unloadEventEnd</dfn> attribute MUST
-        return a {{DOMHighResTimeStamp}} with a
-        time value equal to the time immediately after the user agent
-        finishes the <a data-cite="HTML/browsing-the-web.html#unloading-documents">unload event</a> of
-        the previous document.
-      </p>
-      <p>
-        If there is no previous document or the above <a data-cite="resource-timing-2#dfn-timing-allow-check">timing
-          allow check</a>
-        algorithm does not pass, this attribute MUST return a <a
-          data-cite="hr-time-2#dom-domhighrestimestamp">DOMHighResTimeStamp</a> with a
-        time value equal to zero.
-      </p>
-      <p data-dfn-for='PerformanceNavigationTiming'>
-        The <dfn>domInteractive</dfn> attribute MUST return a <a
-          data-cite="hr-time-2#dom-domhighrestimestamp">DOMHighResTimeStamp</a> with a
-        time value equal to the time immediately before the user agent sets
-        the <a data-cite="HTML/dom.html#current-document-readiness">current
+        <p data-dfn-for='PerformanceNavigationTiming'>
+          When getting the value of the <dfn>unloadEventStart</dfn> attribute,
+          run the following steps:
+          <ul>
+            <li>If there is no previous document, or if the <a>same-origin
+              check</a> fails, return a {{DOMHighResTimeStamp}} with a time
+              value equal to zero.</li>    
+            <li>Otherwise, return a <a data-cite=
+              "hr-time-2#dom-domhighrestimestamp">DOMHighResTimeStamp</a> with a
+              time value equal to the time immediately before the user agent starts
+              the <a data-cite=
+              "HTML/browsing-the-web.html#unloading-documents">unload event</a> of
+              the previous document.</li>
```

Comment:
nit: missing closing \</ul> (or changed to ol). Seems important to not add indentation to the rest of this section :)

Reply:
:+1: